pkg/util/command.go

```diff
@@ -26,6 +26,16 @@ import (
 )
 
 func StandardEnv(manifest model.Manifest) []string {
+	// Unset those environment variables that are set in a container which causes `make xyz` artifacts
```

Comment:
I agree with the design, but I don't think this function should be modifying state. I think we should either do it in an explicit function early in main(), modify `env` here to drop them (this impacts just a variable, not the real environment state), or have the doc that describes running in container drop them in the script before invoking

Reply:
#3 (update doc to have the `unset`s) is probably the easiest. Should update the doc to use `make shell` instead of calling the script directly (and the update to use `make shell` should be done regardless).
#2 is where I had hoped to go with the change. It's just easier to code up a few unset calls than to remove some map entries as a test.

Reply:
Update to only remove the env vars from the spawned `make` processes.